doc/installation.rst

```diff
@@ -99,7 +99,7 @@ To integrate php-cs-fixer as check into Gitlab-CI, you can use a configuration l
     php-cs-fixer:
       image: ghcr.io/php-cs-fixer/php-cs-fixer:${FIXER_VERSION:-3-php8.3}
       script:
-        php-cs-fixer check --diff --format=txt src
+        php-cs-fixer check --format gitlab src
```

Comment:
Shouldn't it be like this?

```suggestion
        php-cs-fixer check --format=gitlab src
```

Reply:
> shoudn't

why should or should not? both variants are valid.